### ISSUE TITLE
Click and drag to act as Capture Region 

### ISSUE THREAD

**Issue by badabing2005** [greenshot/greenshot#409]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
It would be super great if in `Capture Window` mode, With `Use interactive window capture` setting selected, a click and drag would act as `Capture region`
just click - would act as it is currently.

Best of both worlds, Share-X behaves this way and it is very convenient.

**Comment by Ishmaeel** [comment]
To clarify, it is the _other way around_ with ShareX. When you start a region capture, it starts off like the interactive window capture mode, but you can switch to a region capture by dragging. 

The two functions are merged, so, ShareX does not have a separate interactive window capture command.

**Comment by badabing2005** [comment]
> To clarify, it is the _other way around_ with ShareX. When you start a region capture, it starts off like the interactive window capture mode, but you can switch to a region capture by dragging.
> 
> The two functions are merged, so, ShareX does not have a separate interactive window capture command.

Yes that is correct, the two are merged, so there would not be any need for two shortcuts.
